lib/utils.js

```diff
@@ -57,23 +58,32 @@ async function getAndCheckIosSdkVersion () {
   }
 }
 
-function translateDeviceName (platformVersion, devName = '') {
-  let deviceName = devName;
-  switch (devName.toLowerCase().trim()) {
-    case 'iphone simulator':
-      deviceName = 'iPhone 6';
-      break;
-    case 'ipad simulator':
-      // iPad Retina is no longer available for ios 10.3
-      //   so we pick another iPad to use as default
-      deviceName = platformVersion && util.compareVersions(platformVersion, '<', '10.3') ? 'iPad Retina' : 'iPad Air';
+/**
+ * Get the generic simulator for a given IOS version and device type (iPhone, iPad)
+ *
+ * @param {string|number} platformVersion IOS version. e.g.) 13.0
+ * @param {string} deviceName Type of IOS device. Can be iPhone, iPad (possibly more in the future)
+ */
+function getGenericSimulatorForIosVersion (platformVersion, deviceName) {
+  const genericSimulators = iosGenericSimulators[deviceName];
```

Comment:
what if this returns undefined?

Reply:
I just fixed that now. That did cause an error.